multimedia/MuseScore/MuseScore.SlackBuild

```diff
@@ -101,7 +101,7 @@ export Qt6Test_DIR=/opt/MuseScore-Libraries/lib/cmake/Qt6Test
     -DCMAKE_MODULE_LINKER_FLAGS_RELEASE="-pthread" \
     -DQT_QMAKE_EXECUTABLE=/opt/MuseScore-Libraries/bin/qmake
 
-/opt/cmake-opt/bin/cmake --build b
+/opt/cmake-opt/bin/cmake --build b -j 8
```

Comment:
We don't allow hard coded job counts. You can parse MAKEFLAGS if this really need the j arg passed, but typically most things will respect MAKEFLAGS from the env